### DISCUSSION TITLE
Bug when generate a new password for a branch

### DISCUSSION THREAD

**Discussion by agusescobar** [planetscale/discussion#150] [Bug Reports] [answered]
Hi, how are you? i have a problem, i generate a new branch for my database but when i try to generate a connection the password is replaced by a string of * and i can't connect to my database.
What can be? Why now the password at the time of generating it appears like this? I remember that before it showed the password

![image](https://user-images.githubusercontent.com/42684825/149379654-dce666e8-b086-41e5-a773-ef07db2dc90e.png)

**Comment by tbarn** [comment, marked as answer]
Hey @agusfesc! This is what appears when you have viewed the password once before. After you first create a password, it is are not viewable after you close the modal. We do this to increase the security of passwords to your database. You might see a similar pattern with different API providers. You will have to create a new password to see it again. If you have any issues after you create another new password, please let us know!

**Reply by agusescobar** [reply]
Oh! i understand, have you a planned a functionality to show password previously generated? I think show password with a security step (example: the user click on this functionality and have to complete your password or code sended to the email account, or use 2FA code for this)

**Reply by tbarn** [reply]
I do not think it has been considered (yet). From a usage standpoint, do you find yourself having to create new passwords often to get around this?
